### PR TITLE
Initial implementation of image creation

### PR DESCRIPTION
**Pull request by kostyanf14**
The current implementation has a few limitations, that will be fixed in the next PR
- ~~install engine installs all images (studio and 2 clients), existing studio image will be used unless - **--force option is added**~~
- studio and client VMs will be installed from the one Windows ISO - **client platform will be parsed from filename**
- the corresponding version of HCK/HLK should be downloaded manually - **automatic download will be implemented**
- the corresponding windows ISO should be downloaded manually - **automatic download will be implemented**

### CODE REVIEW

File: lib/engines/hckinstall/hckinstall.rb [hunk 1]
```diff
@@ -15,23 +15,205 @@ class HCKInstall
 
     attr_reader :platform
 
+    PLATFORMS_JSON = 'lib/engines/hcktest/platforms.json'
+    ISO_JSON = 'lib/engines/hckinstall/iso.json'
+
+    ANSWER_FILES = ['autounattend.xml', 'unattend.xml'].freeze
+    CLIENTS = %w[c1 c2].freeze
+
+    STUDIO_INSTALL_TIMEOUT = 3 * 60 * 60 # 3 hour
```
Comment (YanVugenfirer) [inline lib/engines/hckinstall/hckinstall.rb:24]: I think moving those parameters to JSON file is a better idea.

File: lib/engines/hckinstall/hckinstall.rb [hunk 2]
```diff
@@ -15,23 +15,205 @@ class HCKInstall
...
+    CLIENT_INSTALL_TIMEOUT = 1 * 60 * 60 # 1 hour
```
Comment (YanVugenfirer) [inline lib/engines/hckinstall/hckinstall.rb:25]: What's the difference?

**Review by YanVugenfirer — CHANGES_REQUESTED** [review]

File: lib/engines/hckinstall/hckinstall.rb (hunk 2 above)
Comment (kostyanf14) [reply lib/engines/hckinstall/hckinstall.rb:25]: HLK studio required more time for installation. The longest studio installation time was 2 hours.
The longest client installation time was 30 min.

File: lib/engines/hckinstall/hckinstall.rb (hunk 1 above)
Comment (kostyanf14) [reply lib/engines/hckinstall/hckinstall.rb:24]: Will do this

**Comment by kostyanf14** [comment]
Added hckinstall config file with all timeouts and unattended files list. Fixed first limitation (`--force-install` option added)

**Review by Basils — APPROVED** [review]

**Review by YanVugenfirer — APPROVED** [review]

**Pull request merged**